src/org/exist/collections/CollectionCache.java

```diff
@@ -38,13 +39,13 @@
  * 
  * @author wolf
  */
-public class CollectionCache extends LRUCache {
+public class CollectionCache extends LRUCache implements BrokerPoolService {
 
     private Object2LongHashMap names;
     private BrokerPool pool;
 
     public CollectionCache(BrokerPool pool, int blockBuffers, double growthThreshold) {
-        super(blockBuffers, 2.0, 0.000001, CacheManager.DATA_CACHE);
+        super(blockBuffers, 2.0, growthThreshold, CacheManager.DATA_CACHE);
```

Comment:
@shabanovd There is no common stress-test facility for eXist. I could run some stress-tests but they would only be representative of a specific artificial workload, which almost certainly would not be the same as your workload.

I will change this to the previous value of `0.000001`...